src/Sulu/Component/Webspace/Loader/XmlFileLoader.php

```diff
@@ -296,6 +299,39 @@ private function generateTheme()
             $theme->addExcludedTemplate($templateNode->nodeValue);
         }
 
+        $this->generateErrorTemplates($theme);
+
+        return $theme;
+    }
+
+    private function generateErrorTemplates(Theme $theme)
+    {
+        $errorTemplates = 0;
+        $defaultErrorTemplates = 0;
+
+        foreach ($this->xpath->query('/x:webspace/x:theme/x:error-templates/x:error-template') as $errorTemplateNode) {
+            /** @var \DOMNode $errorTemplateNode */
+            $errorTemplates++;
+            if (($codeNode = $errorTemplateNode->attributes->getNamedItem('code')) !== null) {
+                $code = $codeNode->nodeValue;
+            } elseif (($defaultNode = $errorTemplateNode->attributes->getNamedItem('default')) !== null) {
+                $default = $defaultNode->nodeValue === 'true';
+                if (!$default) {
+                    throw new InvalidDefaultErrorTemplateException();
```

Comment:
What does this exception say at all? Is it illegal to have a node with the default value of false?